### PR TITLE
Feature/fetch api data

### LINKED ISSUE #5: [Iteration 3] When the server returns a 500 error, the user will see proper error handling
**Please Describe The Problem To Be Solved**
When the server returns a 500 error, the user will see proper error handling

Acceptance Criteria: 
- Ensure APIs are functioning and fetched data is showing up on the page.
- Add error message to screen, so the user knows the server is down.

### PR DESCRIPTION
**Pull request by jfarelli**
# What does this PR do?
- [x] New feature (adds functionality)
- [] Debugging/Refactor

**Implementation/Changes:**
* Added `apiCalls.js` file to house dynamic fetch functionality.
* Added error handling to fetch functionality in case the server is down.
* Also, conditionally rendered a message on the page, so the user is notified if the server is down.
* Imported the function into the `App` file and created `componentDidMount` functionality to pull in the API data.
* Further styling with css of each `Nav`, `MovieCard`, & `MovieCardsCollection`
* Added a hidden button to the `Nav`, so when the user is viewing a single movie's details, they can return to the home page ( will probably need to be deleted later, once we get Router working ). 
* **Button has only been created and is non-functional at the moment. We will need to create functionality to return to the main page.**

**Where should your reviewer start?**
* All files mentioned above.

**What (if any) features are you implementing?**
* Fetching data from the API.

**What issues does this PR close?**
* Closes #5 , 

**Is there anything that you need from your teammate?**
* Please review and give any feedback.

# How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Ant-Shell** [comment]
Awesome, thanks again for walking me through this!

File: src/apiCalls.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const fetchData = ( pathAddress ) => {
+    return fetch( `https://rancid-tomatillos.herokuapp.com/api/v2/${ pathAddress }` )
```
Comment (nicktu12) [inline src/apiCalls.js:2]: Nice job making this function dynamic!

File: src/components/MovieCardsCollection/MovieCardsCollection.js [hunk 2]
```diff
@@ -14,6 +15,7 @@ const MovieCardsCollection = ( { individualMovieCard } ) => {
   return (
     <div className="movie-card-collection">
       { movieCard }
+      { !movieCards.length && <h1 className='error-message'>Our Appologies, but our servers are temorarily down. Plesae try again later.</h1> }
```
Comment (nicktu12) [inline src/components/MovieCardsCollection/MovieCardsCollection.js:18]: Could a ternary be used here to combine lines 17/18 succinctly?

File: src/components/MovieCardsCollection/MovieCardsCollection.css [hunk 3]
```diff
@@ -3,5 +3,13 @@
     grid-template-columns: repeat( 5, 1fr );
     padding: 20px;
     gap: 20px;
-    
+}
+
+.error-message {
+    font-family: 'Italiana', serif;
```
Comment (nicktu12) [inline src/components/MovieCardsCollection/MovieCardsCollection.css:9]: Are other fonts being used, and could this CSS rule be applied globally if so?

**Pull request merged**